### PR TITLE
fixtodo:validate events on PVCs in integration volume binding test

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
fixtodo:validate events on PVCs in integration volume binding test
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/integration/scheduler/volume_binding_test.go [hunk 1]
```diff
@@ -571,6 +583,25 @@ func validatePVCPhase(t *testing.T, client clientset.Interface, pvcName string,
...
+		t.Errorf("Failed to get PVC %v/%v: %v", ns, pvcName, err)
+	}
+
+	pvcEvents, err := client.CoreV1().Events(ns).Search(legacyscheme.Scheme, claim)
+	if err != nil {
+		t.Errorf("Failed to get PVC events %v/%v: %v", ns, pvcName, err)
+	}
+
+	for _, e := range pvcEvents.Items {
+		if e.Reason != reason {
```
[the remark this answers is not in the document]
Comment (hanxiaoshuai) [reply test/integration/scheduler/volume_binding_test.go]: Done, Thanks for your review

File: test/integration/scheduler/volume_binding_test.go [hunk 2]
```diff
@@ -571,6 +583,22 @@ func validatePVCPhase(t *testing.T, client clientset.Interface, pvcName string,
...
+	if err != nil {
+		t.Errorf("Failed to get PVC %v/%v: %v", ns, pvcName, err)
+	}
+
+	pvcEvents, err := client.CoreV1().Events(ns).Search(legacyscheme.Scheme, claim)
+	if err != nil {
+		t.Errorf("Failed to get PVC events %v/%v: %v", ns, pvcName, err)
+	}
+
+	if len(pvcEvents.Items) > 0 && pvcEvents.Items[0].Reason != reason {
```
Comment (msau42) [inline test/integration/scheduler/volume_binding_test.go]: I think it's possible for PVCs to have multiple events.  So I think we should iterate through all the events and find one that matches.

Comment (hanxiaoshuai) [reply test/integration/scheduler/volume_binding_test.go]: Done

**Review by bsalamat — COMMENTED** [review]
/lgtm

**Pull request merged**